.pre-commit-config.yaml

```diff
@@ -6,3 +6,13 @@ repos:
     hooks:
     - id: black
       language_version: python3.7
+  - repo: https://github.com/pycqa/flake8
+    rev: 4.0.1
+    hooks:
+    - id: flake8
+      language_version: python3.7
+  - repo: https://github.com/asottile/reorder_python_imports
+    rev: v3.3.0
+    hooks:
+    - id: reorder-python-imports
+      language_version: python3.7
```

Comment:
I'd rather we didn't use reorder_python_imports. It fundamentally changes the styling used for imports in this library, and is not as 'standard' as black or flake.

On a personal opinion level I also strongly dislike breaking imports from the same modules into separate items and consider
```
from flask import g, request
```
much more readable than
```
from flask import g
from flask import request
```